src/javaRosa.js

```diff
@@ -792,7 +792,12 @@ define([
     };
 
     var itextMediaBlock = function (mug, options) {
-        var block = itextConfigurableBlock(mug, options);
+        var block = itextConfigurableBlock(mug, options),
+            pathPrefix = options.pathPrefix;
+
+        if(!_.isString(options.pathPrefix)) {
```

Comment:
I wouldn't ever expect a basic user to detect any of the bugs we're talking about. However, many other values (like an object or array) would be glaringly obvious to a developer.